doc/news/changes/minor/20170322VishalBoddu

```diff
@@ -0,0 +1,7 @@
+New: GridTools::compute_bounding_box computes a bouding box of a subdomain whose active cells conform to a given predicate. 
+GridTools::compute_active_cells_within_skin computes a collection of active cells that are within a skin_thickness distance from 
```

Comment:
i would call this one `compute_active_cells_within_distance`, but let's see what others think.

"active cells that are within a given distance from" (removed skin_thickness)